docs/templates/plugin.rst.j2

```diff
@@ -116,11 +116,13 @@ Parameters
                     <td class="elbow-placeholder"></td>
                 {% endfor %}
                 {# parameter name with required and/or introduced label #}
-                <td colspan="@{ from_kludge_ns('maxdepth') - loop.depth0 }@">
+                <td colspan="@{ from_kludge_ns('maxdepth') - loop.depth0 }@" style="position: relative">
```

Comment:
What do you need `position: relative` for? I don't see any `position: absolute` below.

Reply:
I guess it is a leftover from attempts to get the "added in version" to the bottom of the cell.
And that seems to be impossible, so I left that idea.

Reply:
Fixed.